business-services/dashboard-analytics/src/main/java/com/tarento/analytics/dto/AggregateDto.java

```diff
@@ -92,4 +92,14 @@ public Object getFilter() {
 	public void setFilter(Object filter) {
 		this.filter = filter;
 	}
+
+//	private long total;
```

Comment:
why has this commented code been added? This needs to be removed.

Reply:
this is pagination changes that were implemented but are not required now, so stored that for backup if will require it in the future we can use that.